README.md

```diff
@@ -9,24 +9,33 @@ This site is written using [Jekyll](https://jekyllrb.com/), a static site genera
 ## Local Development
 
 #### Prerequisites:
-1. [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
-2. [Bundler](https://bundler.io/) (`gem install bundler`)
+
+1. [Docker](https://docker.com) or [Podman](https://podman.io)
 
 #### Instructions
-1. `bundle install`
-2. Add necessary environment variables (they MUST be defined!!!):
-	* `export CSHPUBSITE_ASSETS_URL="https://assets.csh.rit.edu/pubsite"`
-	* `export CSHPUBSITE_S3_URL="https://s3.csh.rit.edu"`
-3. `bundle exec jekyll serve`
 
-The site should now serve locally at `localhost:4000`, and auto-build when you change/create files in the repo!
+First, run the following command in the directory containing the Dockerfile to build the container
+
+```
+podman build -t public-site .
+```
+
+Next, run the following command to start the container
+
+```
+podman run -p 8080:80 public-site:latest
```

Comment:
4000 is intentionally used as a dev port as it's outside the restricted range. It's also likely specifically registered as a port SSO will allow for this client

Reply:
8080 is probably fine, but might need configured in sso

Reply:
It has already been changed by @Mstrodl but documentation hadnt been updated yet

Reply:
Also: the pubsite doesn't need SSO...